pkg/user/spam_txclient/README.md

```diff
@@ -0,0 +1,18 @@
+# TxClient Stress Test
+
+This tool repeatedly broadcasts blob transactions to **Mocha node** with short TTLs and monitors the Celestia TxClient to ensure it never halts.
```

Comment:
[question] historically we've put most tools in ./tools so I wonder if this should be relocated outside `pkg/user`